code/modules/xenomorph/xeno_structures.dm

```diff
@@ -44,6 +43,7 @@
 
 /obj/structure/resin/silo/Initialize()
 	. = ..()
+	resistance_flags |= DROPSHIP_IMMUNE
```

Comment:
ew. Just add all the correct resistance flags at compilation, don't do this

Reply:
yes just have them declared properly

Reply:
> Just add all the correct resistance flags at compilation

As there are flags set for a parent it was that kind of thinking that caused the silos to lose their UNACIDABLE in the first place. Pretty sure my way was a fine practice at some point.

I lack faith in future contributors/reviewers to not make the same mistake by forgetting to check for parent flags and I do not think taking that risk is worth the performance gain of not changing a bitmask at runtime, but have it your way.